src/api.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
```

Comment:
I'd pop this file in a utils folder rather than at the root of your directory. 
You might create more util functions... 🤔